interfaces/builtin/core_support.go

```diff
@@ -19,7 +19,7 @@
 
 package builtin
 
-const coreSupportSummary = `special permissions for the core snap`
+const coreSupportSummary = `formerly special permissions for the core snap`
```

Comment:
Let's please keep the original summary. This is still holding special permissions for core, and it's okay that at the moment core needs none.

Reply:
Ha, true :)